apis/v1alpha1/shared_types.go

```diff
@@ -196,6 +196,21 @@ type AdminNetworkPolicyEgressPeer struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=25
 	Networks []CIDR `json:"networks,omitempty"`
+	
```

Comment:
@astoycos @npinaeva @tssurya 

Our NPEP says we only support ANP, not BANP. Does that mean we should create two different EgressPeer types for ANP and BANP? 😢

Reply:
I think we'll have to unfortunately :/ @tssurya can chime in if there's another way since she implemented it in the first place

I dont think it's too big of a deal though, we would just move the `EgressPeer` typedef out of shared_types.go

Reply:
@rahulkjoshi You should be good to go ahead and make this change :)

Reply:
It's done! PTAL